understanding/20/use-of-color.html

```diff
@@ -38,13 +38,26 @@ <h2>Intent of Use of Color</h2>
          only in color.
       </p>
       
-      <p>Examples of information conveyed by color differences: “required fields are red",
-         “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples
-         of indications of an action include: using color to indicate that a link will open
-         in a new window or that a database entry has been updated successfully. An example
-         of prompting a response would be: using highlighting on form fields to indicate that
-         a required field had been left blank.
-      </p>
+      <p>Examples of common failures include:</p>
+      <ul>
+        <li><strong>Information conveyed by color differences:</strong> required fields are only identified
+         by changing their border to a red color; sales figures for different employees are only
+         visually distinguished by using different foreground colors( e.g. Mary's sales are in red,
+         Tom's are in blue).</li>
```

Comment:
```suggestion
        <li>
            <strong>Information conveyed by color differences:</strong> 
            <ul>
                <li>required fields are only identified by changing their border to a red color;</li>
                <li>the company transactions are only visually distinguished by using different foreground colors: red for debits and green for credits;</li>
                <li>subway lines are visually distinguished solely by employing various filled colors.</li>
            </ul>
        </li>
```

Not essential, but I've proposed a slightly different example, replacing the scenario involving Mary and Tom with a more common one, where credit and debit transactions are distinguished solely by colors. 

Additionally, I've included another common example, where subway lines are identified only by colors. 

As for the WCAG policy on nested lists, I'm uncertain. When there's only one nested list, it often feels more organized to me, but I'm open to using multiple sentences if needed and change my suggestion.